clang/lib/Driver/ToolChains/Arch/LoongArch.cpp

```diff
@@ -251,6 +251,20 @@ void loongarch::getLoongArchTargetFeatures(const Driver &D,
     } else /*-mno-lasx*/
       Features.push_back("-lasx");
   }
+
+  // Select frecipe feature determined by -m[no-]frecipe.
+  if (const Arg *A =
+          Args.getLastArg(options::OPT_mfrecipe, options::OPT_mno_frecipe)) {
+    // FRECIPE depends on 64-bit FPU.
```

Comment:
Why doesn't `frecipe.s` work with only `+f`?

Reply:
My Bad. I mistakenly thought that the LoongArch v1.1 instructions required fpu64 like lsx.